lib/rex/proto/kerberos/model/pkinit/name.rb

```diff
@@ -0,0 +1,24 @@
+# -*- coding: binary -*-
+
+module Rex
+  module Proto
+    module Kerberos
+      module Model
+        module Pkinit
+          class Name
+            attr_accessor :value
```

Comment:
It isn't required, no. This turned out to be an annoying quirk of `RASN1`. Because it uses reflection and such to build up its structure, it does so differently depending on whether the type in question exists within a sequence or on its own. In these cases, rather than manually specifying the entire structure (i.e. creating a Ruby class for every property of a certificate, and their sub-properties, and the sub-properties' sub-sub-properties etc. etc.), I'm just passing off to the OpenSSL implementation of Certificate and Name to do the parsing and `to_der`ring; thus why these two behave a little different from the rest.